Made a new switchable theme button

Mysticpasta1 opened the pull request:
## Purpose
_Describe the problem or feature in addition to a link to the issues when possible._
Trying to make themes and switch between them

## Approach
_How does this change address the problem?_
So far I have made a switchable button, and I hope to switch themes with this button.

#### Open Questions and Pre-Merge TODOs
- [ ] Use github checklists. When solved, check the box and explain the answer.

## Learning
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

#### Blog Posts
- [A Complete Guide to useEffect](https://overreacted.io/a-complete-guide-to-useeffect/) Learn how to use useeffect.

Mysticpasta1 committed:
Made a new switchable theme button

it doesn't switch themes yet

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/gorilla-devs/GDLauncher?pullRequest=1402) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/gorilla-devs/GDLauncher?pullRequest=1402) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/gorilla-devs/GDLauncher?pullRequest=1402) it.</sub>

ExoPlant commented:
May want to set it o draft @Mysticpasta1

Mysticpasta1 committed:
Make it prettier

Mysticpasta1 committed:
The Good Stuff

Name change for button variables (button might be removed in the future)
Customizable Dynamic theme (unuse atm) stuff  based on https://ant.design/docs/react/customize-theme-variable#How-to-use
